hazelcast/pom.xml

```diff
@@ -322,20 +322,21 @@
             <groupId>net.sourceforge.findbugs</groupId>
             <artifactId>annotations</artifactId>
             <version>1.3.2</version>
-            <scope>compile</scope>
+            <scope>provided</scope>
         </dependency>
 
         <dependency>
             <groupId>com.eclipsesource.minimal-json</groupId>
             <artifactId>minimal-json</artifactId>
             <version>0.9.2</version>
-            <scope>compile</scope>
+            <scope>provided</scope>
```

Comment:
can't you explicitly specify artefacts you want to shade in regardless of theirs scope?